pkg/machine/applehv/vfkit.go

```diff
@@ -39,6 +39,15 @@ func getDefaultDevices(mc *vmconfigs.MachineConfig) ([]vfConfig.VirtioDevice, *d
 		return nil, nil, err
 	}
 	devices = append(devices, disk, rng, serial, readyDevice)
+
+	if mc.Rosetta {
+		rosetta, err := vfConfig.RosettaShareNew(define.MountTag)
+		if err != nil {
+			return nil, nil, err
+		}
```

Comment:
Fwiw, you can also set `rosetta.InstallRosetta` to `true`, this tells macOS to automatically install rosetta if it's missing (with user prompting).

Reply:
Thank you for your comment! I will uninstall Rosetta in my environment and test it.

Reply:
@cfergeau 
I tested it. 
I did this instead of `vfConfig.RosettaShareNew`. Because of type mismatch.
Is there any other better way to write this?
```
		rosetta := &vfConfig.RosettaShare{
			DirectorySharingConfig: vfConfig.DirectorySharingConfig{
				MountTag: define.MountTag,
			},
			InstallRosetta: true,
		}
```

Reply:
@cfergeau Fixed in https://github.com/containers/podman/commit/f0896648a82375bf0548de8801faa9138dda8207
Please check.

Reply:
> I did this instead of `vfConfig.RosettaShareNew`. Because of type mismatch.
Is there any other better way to write this?

I'll probably change `RosettaShareNew` return value in the next vfkit release to avoid this kind of issues.
An alternative is 
```
dev := vfConfig.RosettaShareNew(..)`
rosetta, ok := dev.(*vfConfig.RosettaShare)
if !ok {
...
}
```

But the way you wrote it won't break if I make the API change mentioned before, so probably best to go with it for now.